client/src/views/Report/SunBurst.js

```diff
@@ -129,79 +140,81 @@ export default function SunBurst(props) {
-                                    key={d.key}
-                                    fill={getCellColor(d.key, colorScheme)}
-                                    onClick={() => updateFilterParams('first', d.key)}
-                                    className={styles.sunburstCell}
-                                />
-                            ))
-                        }
-                    </Pie>
-
-                )
-            }
-            {
-                secondLevelData && (
-                    <Pie
-                        data={secondLevelData}
-                        dataKey="size"
-                        innerRadius={centerPieData.length > 0 ? 62 : 42}
-                        outerRadius={90}
-                    >
-                        {
-                            secondLevelData.map(d => (
-                                <Cell
-                                    key={d.key}
-                                    fill={getCellColor(d.key, colorScheme)}
-                                    onClick={() => updateFilterParams('second', d.key)}
-                                    className={styles.sunburstCell}
-                                />
-                            ))
-                        }
-                    </Pie>
-                )
-            }
-            <Tooltip content={<CustomTooltip />} />
-        </PieChart>
+            <PieChart>
+                {
+                    centerPieData && (
+                        <Pie
+                            data={centerPieData}
+                            dataKey="size"
+                            outerRadius={30}
+                        >
+                            {
+                                firstLevelData.map(d => (
+                                    <Cell
+                                        key={d.key}
+                                        fill="#a6cee3"
+                                        onClick={() => updateFilterParams('zero', '')}
+                                        className={styles.sunburstCell}
+                                    />
+                                ))
+                            }
+                        </Pie>
+                    )
+                }
+                {
+                    firstLevelData && (
+                        <Pie
+                            data={firstLevelData}
+                            dataKey="size"
+                            innerRadius={centerPieData.length > 0 ? 32 : 0}
+                            outerRadius={centerPieData.length > 0 ? 60 : 40}
+                        >
+                            {
+                                firstLevelData.map(d => (
+                                    <Cell
+                                        key={d.key}
+                                        fill={getCellColor(d.key, colorScheme)}
+                                        onClick={() => updateFilterParams('first', d.key)}
```

Comment:
Use callback function for this

Reply:
getCellColor is defined outside the Component

Reply:
callback used for updateFilterParams